examples/cells_output.ts

```diff
@@ -31,7 +31,7 @@
 import * as helpers from "../auraescript/gen/helpers.ts";
 import * as runtime from "../auraescript/gen/cells.ts";
 
-let cells = new runtime.CellServiceClient();
+let cells = new runtime.CellServiceClient("./pki/ca.crt", "./pki/_signed.client.nova.crt", "./pki/client.nova.key", "127.0.0.1:8080");
```

Comment:
given all the clients would want to use the same certs+socket (right?) i think this should be a higher level API ... otherwise we'll be passing the same thing to every client.